src/tests/ftest/deployment/ior_per_rank.yaml

```diff
@@ -36,13 +35,11 @@ ior:
   test_file: testFile
   write_flags: "-w -C -e -g -G 27 -k -Q 1"
   read_flags: "-r -R -C -e -g -G 27 -k -Q 1"
-  sw_deadline: 30
+  sw_deadline: 15
   sw_wearout: 1
   sw_status_file: "/var/tmp/daos_testing/stoneWallingStatusFile"
   dfs_oclass: 'SX'
-  transfer_sizes:
-    - 1M
-    - 256B
+  transfer_size: 1M
```

Comment:
Please update the wiki page: https://daosio.atlassian.net/wiki/spaces/DAOS/pages/11136040981/Running+Rack+Group+Level+Tests
This will have reference to 256B.

Reply:
Will do. I will update the wiki after this merges